bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -103,6 +103,13 @@ public interface Image extends Component {
      */
     String PN_DISPLAY_POPUP_TITLE = "displayPopupTitle";
 
+    /**
+     * Name of the resource property that will indicate if the image component should render an external image resource.
```

Comment:
Isn't it the name of the property holding the path of the external image?

Reply:
No, it's holding the path to the other image component resource

Reply:
I think we mean the same :) . I'd rephrase it as follows:
Name of the property holding the path of the external image resource.